tests/zfs-tests/cmd/libzfs_input_check/libzfs_input_check.c

```diff
@@ -708,7 +708,8 @@ test_redact(const char *snapshot1, const char *snapshot2)
 
 	strncpy(bookmark, snapshot1, sizeof (bookmark) - 1);
```

Comment:
This is right, but it's also the only use of `strncpy` in this source file.  For consistency it would be nice to change it to `strlcpy()` while you're here.

Reply:
changed to strlcpy(3).